test/test_flake8.py

```diff
@@ -18,8 +18,8 @@
 import sys
 
 # flake8 doesn't support Python < 2.7 anymore
-# flake8 3.5.0 does't work on Python 3.2, skip it as well
-if (sys.version_info[0] > 2 or sys.version_info[1] >= 7) and not (sys.version_info[0] == 3 and sys.version_info[1] == 2):
+# flake8 3.5.0 does't work on Python 3.2 and 3.3, skip them as well
+if (sys.version_info[0] > 2 or sys.version_info[1] >= 7) and not (sys.version_info[0] == 3 and sys.version_info[1] <= 3):
```

Comment:
The current state doesn't test with Python 3.3 anymore. So no need for the special handling anymore.

Reply:
Done.